content/en/metrics/faq/update-to-query-value-widget.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Update to Distribution Metrics Workflow FAQ
+kind: faq
+is_beta: true
+further_reading:
+- link: "/metrics/faq/update-to-query-value-widget"
+  tag: "FAQ"
+  text: "Update to Top Lists and Query Value Widgets"
+---
+
+## What is changing?
+
+We have unified the way metric queries are evaluated for Top Lists and Query Value Widgets such that you will have more consistent results for the same metric viewed in a Top List and a Query Value Widget. 
```

Comment:
This feels like a release note, not documentation.

Reply:
```suggestion
Datadog is unifying the way metric queries are evaluated for Top Lists and Query Value widgets so that you get more consistent results for the same metric viewed in a Top List and a Query Value Widget. 
```